### ISSUE TITLE
WakaTime has an unknown error on Linux ARM.

### ISSUE THREAD

**Issue by kallydev** [wakatime/vscode-wakatime#210]
When I work on Linux ARM via Remote SSH, the WakaTime plugin does not work properly. After initialization, as long as I enter any code, it will prompt `WakaTime Error`. 

Operating System:

```text
Linux raspberry-pi 5.4.0-1038-raspi #41-Ubuntu SMP PREEMPT Thu Jun 17 14:14:11 UTC 2021 aarch64 aarch64 aarch64 GNU/Linux
```

Error message：

```text
WakaTime: Unknown Error (2); Check your /home/kallydev/.wakatime.log file for more detail.
```

However, the `.wakatime.log` file does not exist, only `.wakatime` directory and `.wakatime.cfg` file.

I tried the solutions provided by #134 and [Reddit](https://www.reddit.com/r/WakaTime/comments/ke5vvr/working_waka_on_vsc_insiders_mb_m1/). After I set `standalone = false` and tried to restart VS Code and reinstall the plugin, it didn't work.

**Comment by gandarez** [comment]
Hi @kallydev 

What if you use the new cli, with this line added to your ~/.wakatime.cfg file?

```
legacy_python_cli = false
```

Then restart VS Code and let me know if it uses the `wakatime-cli-linux-arm`binary and if it's uploading activity correctly?

**Comment by alanhamlett** [comment]
We just released [v13.0.0](https://github.com/wakatime/vscode-wakatime/blob/master/CHANGELOG.md#1300-2021-07-01) of the plugin that uses the new Go wakatime-cli, so `legacy_python_cli = false` is no longer needed.

**Comment by kallydev** [comment]
> We just released [v13.0.0](https://github.com/wakatime/vscode-wakatime/blob/master/CHANGELOG.md#1300-2021-07-01) of the plugin that uses the new Go wakatime-cli, so `legacy_python_cli = false` is no longer needed.

Aah, v13.0.0 is working fine, the plugin successfully downloaded the Golang version of wakatime-cli from AWS via my local HTTP proxy, and also successfully submitted the heartbeat package. 

> Hi @kallydev
> 
> What if you use the new cli, with this line added to your ~/.wakatime.cfg file?
> 
> ```
> legacy_python_cli = false
> ```
> 
> Then restart VS Code and let me know if it uses the `wakatime-cli-linux-arm`binary and if it's uploading activity correctly?

I also tested v12.0.0, `legacy_python_cli` doesn't seem to work. It has a long initialization process and only prompts `WakaTime: Initialized` at the end, and `.wakatime` folder is empty. I have used a local HTTP proxy service, and it successfully obtained the latest version information through the GitHub API, but there was no download request. 

The problem was solved, thanks for your help! I will recommend WakaTime to my friends, thanks again.
